torch/autograd/_functions/tensor.py

```diff
@@ -591,6 +591,16 @@ def backward(self, grad_output):
         return grad_input
 
 
+def reverse_cumsum(x, dim):
```

Comment:
https://en.wikipedia.org/wiki/Prefix_sum - cumsum is an inclusive scan using +, this function is an exclusive scan using +.